proto-wine/ftnoir_protocol_wine.cpp

```diff
@@ -71,15 +70,16 @@ module_status wine::initialize()
         if (s.proton_appid == 0)
             return error(tr("Must specify application id for Proton (Steam Play)"));
 
-        QProcessEnvironment make_steam_environ(const QString& proton_path, int appid);
+        std::tuple<QProcessEnvironment, QString, bool> make_steam_environ(const QString& proton_path, int appid);
         QString proton_path(const QString& proton_path);
 
         wine_path = proton_path(s.proton_path().toString());
-        try {
-            env = make_steam_environ(s.proton_path().toString(), s.proton_appid);
-        } catch(const ProtonException &e) {
-            return error(e.getMessage());
-        }
+        qDebug() << s.proton_path().toString();
+        auto [proton_env, error_string, success] = make_steam_environ(s.proton_path().toString(), s.proton_appid);
+        env = proton_env;
```

Comment:
i tried using 
```
auto [env, error_string, success] = 
```

but i think because env is already declared on line 66, it was being lost when we exited the if scope.

the best solution i could find was to give it a different name and assign it to `env`

Reply:
Either that or use `std::tie`.